packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -895,8 +894,8 @@ describe('Profiler', () => {
             // But it spends time rendering all of the failed subtree also.
             expect(mountCall[1]).toBe('mount');
             // actual time includes: 2 (ErrorBoundary) + 5 (AdvanceTime) + 10 (ThrowsError)
-            // If replayFailedUnitOfWorkWithInvokeGuardedCallback is enbaled, ThrowsError is replayed.
-            expect(mountCall[2]).toBe(flagEnabled && __DEV__ ? 27 : 17);
+            // We don't count the time spent in replaying the failed unit of work (ThrowsError)
+            expect(mountCall[2]).toBe(17);
```

Comment:
This particular detail seems kind of arbitrary.

The PR changes the timing so that we override the actual time spent during the failure with the time spent while retrying it (only for that particular fiber)– but I think this is fine.